http/logger.go

```diff
@@ -28,14 +27,14 @@ type logEntry struct {
 var _ middleware.LogEntry = (*logEntry)(nil)
 
 func (e *logEntry) Write(status, bytes int, header http.Header, elapsed time.Duration, extra any) {
-	log.Info(
+	log.InfoContext(
 		e.req.Context(),
 		"Request",
-		logging.NewKV("Method", e.req.Method),
-		logging.NewKV("Path", e.req.URL.Path),
-		logging.NewKV("Status", status),
-		logging.NewKV("LengthBytes", bytes),
-		logging.NewKV("ElapsedTime", elapsed.String()),
+		"Method", e.req.Method,
```

Comment:
todo: These no longer look 'structured' looking at the call `"Method": methodValue, "Path": pathValue, etc`, instead it looks like they would be logged as equal and independent values `"Method", methodValue, "Path", pathValue, ...`?

I think that is a loss, and one worth chatting about with the team.

Reply:
It is possible to use `slog.String("key", "val")` or any of the other variants here in place of the alternating key value pairs. I have a slight preference for the alternating key values but happy to hear more opinions.

Reply:
I'm also in favour of using `slog.String("key", "val")`. Is there a reason for your preference of the alternating key value pairs that could help change our minds?

Reply:
> Is there a reason for your preference of the alternating key value pairs that could help change our minds?

It's easier for me to read, but I don't mind the other option.

Reply:
> It's easier for me to read, but I don't mind the other option.

Might be worth chatting with the protocol (and the rest of the db team) about this - whilst the current interface allows for both to be used, we might want to force `slog.Foo("Key", val)` like the old logging package used to.